src/main/java/seedu/duke/commands/EditLessonCommand.java

```diff
@@ -60,9 +63,9 @@ public void execute(UI ui) {
     /**
      * Obtains the fields to be edited from the user and makes the changes
      * to the Lesson object.
-     * 
+     *
      * @param lesson the lesson to be edited
-     * @param ui instance of UI for printing prompts
+     * @param ui     instance of UI for printing prompts
      */
     private void editLessonFields(Lesson lesson, UI ui) {
```

Comment:
I still feel that we can move some parts that involve editing the Lesson object to the Lesson class itself so that we don't have to pass the object around just to edit its attributes. But I agree that printing/prompting (UI methods)/etc should be done here in EditLessonCommand. Like what @isaharon said too.

Similarly, I have taken note of this for EditTaskCommand (making changes now).